### PR TITLE
MAINTAINERS - Add Matilda Clerke

### PR DESCRIPTION
**Pull request by Gabriel-Trintinalia**
I propose to add @Matilda-Clerke  as a Besu project maintainer.
@Matilda-Clerke  is making high-quality contributions to besu, focusing on improving the Peer Task management and  the new approach for Peertask system. 

A sample:
- https://github.com/hyperledger/besu/pull/7638
- https://github.com/hyperledger/besu/pull/7628
- https://github.com/hyperledger/besu/pull/7633
- https://github.com/hyperledger/besu/pull/7512
- https://github.com/hyperledger/besu/pull/7356
- https://github.com/hyperledger/besu/pull/7472

Full list of PRs https://github.com/hyperledger/besu/pulls?q=is%3Apr+author%3AMatilda-Clerke+is%3Aclosed

Voting ends two weeks from today.
For more information on this process see the Becoming a Maintainer section in the MAINTAINERS.md file.

### CODE REVIEW

**Comment by Matilda-Clerke** [comment]
Thank you for nominating me to be a maintainer. I'm looking forward to contributing to Besu for a long time!

**Review by fab-10 — APPROVED** [review]

**Review by daniellehrner — APPROVED** [review]

File: MAINTAINERS.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 | Lucas Saldanha   | lucassaldanha    | lucassaldanha    |
 | Sally MacFarlane | macfarla         | macfarla         |
 | Karim Taam       | matkt            | matkt            |
+| Matilda Clerke   | Matilda-Clerke   | MatildaClerke    |
```
Comment (macfarla) [inline MAINTAINERS.md:26]: nit: alphabetical sort means it should be one line above

**Review by macfarla — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (Gabriel-Trintinalia) [reply MAINTAINERS.md:25]: Thanks

**Review by siladu — APPROVED** [review]

**Review by gfukushima — APPROVED** [review]

**Comment by Gabriel-Trintinalia** [comment]
Merging as vote period completed two weeks from the proposal date

**Pull request merged**